pom.xml

```diff
@@ -98,7 +98,7 @@
             <compilerArgs>
               <!-- Args related to Error Prone, see: https://errorprone.info/docs/installation#maven -->
               <arg>-XDcompilePolicy=simple</arg>
-              <arg>-Xplugin:ErrorProne</arg>
+              <arg>-Xplugin:ErrorProne -XepExcludedPaths:.*gson/src/test.*|.*extras/src/test.*|.*proto.*</arg>
```

Comment:
Is it intended that you disable warnings for the complete `proto` module? Maybe it would be better to only disable warnings for the generated source code, as suggested in https://github.com/google/gson/pull/2308#discussion_r1096470904.